devices/src/virtio/net.rs

```diff
@@ -162,12 +163,15 @@ struct NetEpollHandler {
 }
 
 impl NetEpollHandler {
-    fn signal_used_queue(&self) {
+    fn signal_used_queue(&self) -> std::result::Result<(), DeviceErr> {
```

Comment:
```suggestion
    fn signal_used_queue(&self) -> result::Result<(), DeviceError> {
```
The same applies for all the following occurrences.